e2e/src/helpers.ts

```diff
@@ -57,8 +58,147 @@ export async function getTeamPage({
   browser: Browser;
   userId: number;
   teamName: string;
-}): Page {
+}): Promise<Page> {
   const page = await getAdminPage({ browser, userId });
   await page.locator("h2", { hasText: teamName }).click();
   return page;
 }
+
+export async function saveSession({
+  page,
+  adminGQLClient,
+  context,
+}: {
+  page: Page;
+  adminGQLClient: GraphQLClient;
+  context: Context;
+}): Promise<string | undefined> {
+  await page
+    .locator("button", { hasText: "Save and return to this application later" })
+    .click();
+  await page.waitForResponse((response) => {
+    return response.url().includes("/send-email/save");
+  });
+  const sessionId = await findSessionId(adminGQLClient, context);
+  return sessionId;
+}
+
+export async function returnToSession({
+  page,
+  context,
+  sessionId,
+}: {
+  page: Page;
+  context: Context;
+  sessionId: string;
+}) {
+  const returnURL = `/${context.team?.slug}/${context.flow?.slug}/preview?analytics=false&sessionId=${sessionId}`;
+  await page.goto(returnURL);
+  await page.locator("#email").fill(context.user?.email);
+}
+
+export async function clickContinue({
+  page,
+  waitForResponse = false,
+}: {
+  page: Page;
+  waitForResponse?: boolean;
+}) {
+  await page.getByTestId("continue-button").click();
+  if (waitForResponse) {
+    await page.waitForResponse(
+      (response) =>
+        response.url().includes("graphql") && response.status() === 200
+    );
+  }
+}
+
+export async function fillInEmail({
+  page,
+  context,
+}: {
+  page: Page;
+  context: Context;
+}) {
+  await page.locator("#email").fill(context.user.email);
+  await page.locator("#confirmEmail").fill(context.user.email);
+}
+
+export async function findQuestion({
+  page,
+  title,
+}: {
+  page: Page;
+  title: string;
+}): Promise<Locator> {
+  return await page.getByRole("group", { name: title });
+}
+
+export async function answerQuestion({
+  page,
+  title,
+  answer,
+}: {
+  page: Page;
+  title: string;
+  answer: string;
+}) {
+  const group = await findQuestion({ page, title });
+  await group.getByRole("button", { name: answer }).click();
+}
+
+export async function answerChecklist({
```

Comment:
I'd be curious to know if this works as-is for expandable checklists - I think the labels are always in the DOM and just hidden when the "parent" is closed. No changes needed - just a thought / comment!